configs/textrecog/nrtr/nrtr_modality_transform_academic.py

```diff
@@ -0,0 +1,32 @@
+_base_ = [
+    '../../_base_/default_runtime.py',
+    '../../_base_/recog_models/nrtr_modality_transform.py',
+    '../../_base_/schedules/schedule_adam_step_6e.py',
+    '../../_base_/recog_datasets/ST_MJ_train.py',
+    '../../_base_/recog_datasets/academic_testset.py',
```

Comment:
```suggestion
    '../../_base_/recog_datasets/academic_test.py',
```
Still a wrong path. Need to run a test before committing.